### DISCUSSION TITLE
Example of using hono.js streaming response for OpenAI on Cloudflare?

### DISCUSSION THREAD

**Discussion by gouku** [honojs/hono#1732] [Q&A]
Hi, I see hono.js has stream support but how to make it works with OpenAI API on Cloudflare? An example would be appreciated.

**Comment by yusukebe** [comment]
Hi @gouku 

You can refer to this:

https://github.com/yusukebe/chatgpt-streaming/blob/21e65db9ffcfb3260a6d541f8318801b2e2dd58f/app/routes/index.tsx#L19-L47

This is using Sonik which wrap Hono, so this code is a almost Hono code.
